cypress/support/reusable_tests/admin.js

```diff
@@ -193,7 +203,209 @@ Cypress.Commands.add('testAdminFilterAddNewManagerAction',(viewport,creds,admin)
                 });
             }
         });
+    });  
+});
+
+Cypress.Commands.add('testAdminTrashBinElements',(viewport,creds)=>{
+    cy.viewport(viewport);
+    cy.login(creds.email,creds.password);
+    
+     //Waiting for Response
+     cy.intercept('/v1/organizations/**');
+
+    
+    //Intercept
+    cy.intercept('/v1/organizations?deleted=true').as('deletedOrganizations');
+    cy.intercept('/v1/organizations?serviceDeleted=true').as('deletedOrgServices');
+    //Click
+    cy.getElementByTestId('header-admin-link').click();
+     
+    cy.wait(['@deletedOrganizations','@deletedOrgServices'])
```

Comment:
I think this wait is in the wrong place. it should go after said actions are invoked e.g. whatever action triggers the network request

Reply:
The cy.login triggers a response. Immediately after the login, the redirect to dashboard triggers the organizations endpoint.

Reply:
Admin Link- routes to endpoint( /admin ), this endpoint does all the calls at once for all 3 available tabs(Users,Suggestions,Trash Bin) under that option. So I have to intercept after that click